libraries/AP_AHRS/AP_AHRS_Backend.cpp

```diff
@@ -90,7 +89,7 @@ Vector2f AP_AHRS_DCM::groundspeed_vector(void)
     if (gotAirspeed) {
         const Vector3f wind = wind_estimate();
         const Vector2f wind2d(wind.x, wind.y);
-        const Vector2f airspeed_vector(_cos_yaw * airspeed, _sin_yaw * airspeed);
+        const Vector2f airspeed_vector{cosf(yaw) * airspeed, sinf(yaw) * airspeed};
```

Comment:
is this being done once per loop or per call?

Reply:
Per-call.  OTOH, there's a method in the same function which was doing the exact same thing, sooo.... :-)

I've added a cache for these values - updated on reset and on update()